# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/xfce4-terminal/default-transparent.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- xfce4-terminal-1.1.3/terminal/terminal-preferences.c.vanilla	2024-05-24 11:59:10.268486023 +0200
+++ xfce4-terminal-1.1.3/terminal/terminal-preferences.c	2024-05-24 11:59:57.433481478 +0200
@@ -307,7 +307,7 @@
                          NULL,
                          "BackgroundMode",
                          TERMINAL_TYPE_BACKGROUND,
-                         TERMINAL_BACKGROUND_SOLID,
+                         TERMINAL_BACKGROUND_TRANSPARENT,
                          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
 
   /**
@@ -343,7 +343,7 @@
       g_param_spec_double ("background-darkness",
                            NULL,
                            "BackgroundDarkness",
-                           0.0, 1.0, 0.5,
+                           0.0, 1.0, 0.666,
                            G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS);
 
   /**
